### PR TITLE
chore: add support arbitrum and optimism

### PR DESCRIPTION
**Pull request by sachushaji**
this change adds support for arbitrum in WRW. Also refactoring things a bit to make it more reusable for EVM Like chains

Ticket: WIN-1118

### CODE REVIEW

File: electron/main/index.ts [hunk 1]
```diff
@@ -36,6 +37,7 @@ import { Erc20Token, Eth, Gteth } from '@bitgo/sdk-coin-eth';
 import { Ethw } from '@bitgo/sdk-coin-ethw';
 import { Ltc } from '@bitgo/sdk-coin-ltc';
 import { Near, TNear } from '@bitgo/sdk-coin-near';
+import {Opeth, Topeth} from '@bitgo/sdk-coin-opeth';
```
Comment (gianchandania) [inline electron/main/index.ts:40]: nit:
```suggestion
import { Opeth, Topeth } from '@bitgo/sdk-coin-opeth';
```

File: src/helpers/config.ts [hunk 2]
```diff
@@ -6,9 +6,11 @@ export type CoinMetadata = {
   Description: string;
   value: string;
   Icon: CryptocurrencyIconProps['Name'];
+  ApiKeyProvider?: string;
+  isTSSSupported?: boolean;
```
Comment (gianchandania) [inline src/helpers/config.ts:10]: nit: isTssSupported. Also shouldn't isTssSupported be added to polygon as well?

File: src/containers/NonBitGoRecoveryCoin/EthLikeForm.tsx [hunk 3]
```diff
@@ -29,6 +30,7 @@ const validationSchema = Yup.object({
 }).required();
 
 export type EthereumFormProps = {
```
Comment (gianchandania) [inline src/containers/NonBitGoRecoveryCoin/EthLikeForm.tsx:32]: I think in this file as well, lets change naming to EthLikeFormProps, etc

File: src/helpers/config.ts (hunk 2 above)
Comment (sachushaji) [reply src/helpers/config.ts:10]: I was seeing only legacy support added for polygon in WRW till now, we could add it but needs someone to test it. Recoveries should work with legacy also

**Review by gianchandania — APPROVED** [review]

**Pull request merged**